### PR TITLE
fix(ui): implement themed delete wallet dialog

### LINKED ISSUE #1191: Implement new Delete Wallet confirmation dialog
## Detailed Description

Implement custom design for delete wallet dialog

## Context

Currently using the system dialog to handle this and the design doesn't really fit with the app.

## Possible Implementation

Implement new Wallet Delete confirmation dialog based on Zeplin design: 
zpl://screen?sid=5bf3f08593b7393e8cf59e13&pid=5bacdbad33af886d419f1b34

image

### PR DESCRIPTION
**Pull request by korhaliv**
fix #1191

### CODE REVIEW

**Comment by korhaliv** [comment]
Fair enough, was just trying to follow the mockup presented. Is there an extended mockup for this ? If not I can add that information the way I see it and update the PR.

**Comment by korhaliv** [comment]
It's actually WIP. This is an old one without new Ole design.

**Comment by korhaliv** [comment]
Good catch. Could barely notice it when testing (prob depends on rendering speed) but can be clearly seen in redux dev tools. Fixed

File: app/preload.js [hunk 1]
```diff
@@ -83,56 +83,32 @@ function killLnd() {
...
+ */
+function getWalletDir(chain, network, wallet) {
+  return join(remote.app.getPath('userData'), 'lnd', chain, network, wallet)
+}
+
 /**
  * Delete a local wallet from the filesystem.
  */
-async function deleteLocalWallet(chain, network, wallet, force = false) {
+async function deleteLocalWallet(chain, network, wallet) {
```
Comment (mrfelton) [inline app/preload.js]: if we are removing the `force` arg, then it should be removed from the `e2e` test suite where it was being used.

File: app/reducers/wallet.js [hunk 2]
```diff
@@ -172,6 +190,18 @@ const walletSelectors = {}
...
 
+walletSelectors.activeWalletDir = createSelector(
+  activeWalletSelector,
+  walletsSelector,
+  (activeWallet, wallets) => {
+    const wallet = wallets.find(w => w.id === activeWallet)
+    if (wallet && wallet.type === 'local') {
+      return window.Zap.getWalletDir(wallet.chain, wallet.network, wallet.wallet)
+    }
+    return ''
```
[the remark this answers is not in the document]
Comment (korhaliv) [reply app/reducers/wallet.js]: done

File: app/preload.js (hunk 1 above)
Comment (korhaliv) [reply app/preload.js]: done

**Pull request merged**